src/collective/importexport/browser/import_view.py

```diff
@@ -29,10 +45,13 @@
 from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile as FiveViewPageTemplateFile
 from collective.z3cform.datagridfield import DataGridFieldFactory, DictRow
 
-import csv
+# import csv
 import logging
 import StringIO
 import time
+import unicodecsv as csv
+import tablib
```

Comment:
Yes, [install_requires](https://packaging.python.org/distributing/#install-requires) contains a list of dependencies, which are pulled automatically when installing with pip or buildout.